### DISCUSSION TITLE
Deny-PublicPaaSEndpoints policy stopping restore of a Cosmos DB account (To a new Account)

### DISCUSSION THREAD

**Discussion by MikaelJcSoderberg** [Azure/Enterprise-Scale#1613] [Q&A]
We have this policy assigned at the Root level, with the default effect 'Deny'

When doing a point in time restore of a Cosmos DB account, when you specify to use a new account, you dont get the oppertunity to create the sub resources that makes the account compliant to this policy.

The team that would do the restore dont have access to create a temporary Exemption to the policy, allowing the account to be created.
Is there any workarounds, any AZ Cli or bicep(ARM) workaround where you can specify more on the account creation that would make this work.

**Comment by jtracey93** [comment]
Please raise this as a support ticket to the CosmosDB team as this sounds like they need to add some options to their portal experience in the restore experience
